packages/applications-service-api/src/controllers/timetables.js

```diff
@@ -7,10 +7,15 @@ const ApiError = require('../error/apiError');
 
 module.exports = {
 	async getTimetables(req, res) {
-		const { caseRef } = req.params;
+		const caseRef = encodeURIComponent(req.params.caseRef);
 
-		logger.debug(`Retrieving timetables by case reference ${caseRef} ...`);
 		try {
+			logger.debug(`Retrieving timetables by case reference ${caseRef} ...`);
+
+			if (caseRef !== req.query.caseRef) {
```

Comment:
Is this meant to be `req.params.caseRef` ?

Reply:
No it's not.  It's the following: 	
	`const caseRef = encodeURIComponent(req.params.caseRef);`

This is one of the requirements for the cross site scripting recommendation highlighted within the ticket.

Reply:
On second thoughts, in the logger.debug it's probably ok for it not to be encoded.  I'll go with your suggestion.

Reply:
Done